packages/node-cli/src/commands/account/manageauth/ManageAccountAction.js

```diff
@@ -70,6 +70,7 @@ module.exports = class ManageAccountAction extends BaseAction {
 			? ManageAccountActionResult.Builder.withData(this._prepareData(selectedOptions, operationResult.data))
 					.withResultMessage(operationResult.resultMessage)
 					.withExecutedAction(selectedOptions.action)
+				    .withCommandParameters(executionContext.getParams())
```

Comment:
It's not aligned with the other lines.

Reply:
I swear it is in intellij, but I will check again :)